package/MDAnalysis/core/groups.py

```diff
@@ -3285,7 +3285,8 @@ def guess_bonds(self, vdwradii=None, fudge_factor=0.55, lower_bound=0.1):
         ----------
         vdwradii : dict, optional
             Dict relating atom types: vdw radii
-
+        context : str
+            context of the universe
```

Comment:
Perhaps add markup to show that this kwarg was added. Check that it renders correctly in the docs, sometimes double-empty lines are needed. (You can also `git grep versionadded` to see how we use it elsewhere.)
```suggestion
            context of the universe

            .. versionadded:: 2.8.0

```